Vagrantfile

```diff
@@ -51,15 +51,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.network "forwarded_port", guest: 9991, host: host_port, host_ip: host_ip_addr
 
   if Vagrant.has_plugin?("vagrant-proxyconf")
-    if http_proxy != ""
       config.proxy.http = http_proxy
-    end
-    if https_proxy != ""
       config.proxy.https = https_proxy
-    end
-    if https_proxy != ""
       config.proxy.no_proxy = no_proxy
-    end
```

Comment:
Can you explain the purpose of this change?  I would think we don't want to set these at all unless we have something to set them to.

I merged the other commit.